File: src/main/java/no/nav/service/pensjon/regler/to/SamletEktefellepensjonRequest.java
package no.nav.service.pensjon.regler.to;


import java.util.ArrayList;
import java.util.List;

import no.nav.domain.pensjon.regler.beregning.Beregning;
import no.nav.domain.pensjon.regler.beregning2011.AbstraktBeregningsResultat;
import no.nav.domain.pensjon.regler.beregning2011.BeregningsResultatAfpPrivat;
import no.nav.domain.pensjon.regler.krav.Kravhode;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

public class SamletEktefellepensjonRequest extends ServiceRequest {
	private AbstraktBeregningsResultat soker1Beregningsresultat;
	private AbstraktBeregningsResultat soker2Beregningsresultat;
	private Beregning soker1Beregning;
	private Beregning soker2Beregning;
	private BeregningsResultatAfpPrivat soker1BeregningsresultatAfpPrivat;
	private BeregningsResultatAfpPrivat soker2BeregningsresultatAfpPrivat;
	private Kravhode soker1Kravhode;
	private Kravhode soker2Kravhode;
	private Kravhode soker1Kravhode2011;
	private Kravhode soker2Kravhode2011;
	private List<VilkarsVedtak> soker1VilkarsvedtakListe = new ArrayList<VilkarsVedtak>();
	private List<VilkarsVedtak> soker2VilkarsvedtakListe = new ArrayList<VilkarsVedtak>();
	private List<VilkarsVedtak> soker1VilkarsvedtakListe2011 = new ArrayList<VilkarsVedtak>();
	private List<VilkarsVedtak> soker2VilkarsvedtakListe2011 = new ArrayList<VilkarsVedtak>();
	 

	
}

File: src/main/java/no/nav/pensjon/regler/to/SamletEktefellepensjonRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning.Beregning
import no.nav.pensjon.regler.domain.beregning2011.AbstraktBeregningsResultat
import no.nav.pensjon.regler.domain.beregning2011.BeregningsResultatAfpPrivat
import no.nav.pensjon.regler.domain.krav.Kravhode
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak

class SamletEktefellepensjonRequest {
    var soker1Beregningsresultat: AbstraktBeregningsResultat? = null
    var soker2Beregningsresultat: AbstraktBeregningsResultat? = null
    var soker1Beregning: Beregning? = null
    var soker2Beregning: Beregning? = null
    var soker1BeregningsresultatAfpPrivat: BeregningsResultatAfpPrivat? = null
    var soker2BeregningsresultatAfpPrivat: BeregningsResultatAfpPrivat? = null
    var soker1Kravhode: Kravhode? = null
    var soker2Kravhode: Kravhode? = null
    var soker1Kravhode2011: Kravhode? = null
    var soker2Kravhode2011: Kravhode? = null
    var soker1VilkarsvedtakListe: List<VilkarsVedtak> = ArrayList()
    var soker2VilkarsvedtakListe: List<VilkarsVedtak> = ArrayList()
    var soker1VilkarsvedtakListe2011: List<VilkarsVedtak> = ArrayList()
    var soker2VilkarsvedtakListe2011: List<VilkarsVedtak> = ArrayList()

    companion object {
        private const val serialVersionUID = 2620310836634304154L
    }
}